mautic/mautic
# Fix campaign loop for jump action for decision

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

If decision has no parent then Jump to action event can make loop in campaing. 



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Make this campaign 

![image](https://user-images.githubusercontent.com/462477/195013642-4c3a7fc2-80ca-41c1-9adc-5e374fcb31fa.png)

3. Run campaign
4. Wait one minute to execute false path of decision
5. Then run again and again and see every time is decision execute to false  and make campaign false

![image](https://user-images.githubusercontent.com/462477/195013822-eb88c435-4618-4a8f-af36-b38cf6651e81.png)
